test/test_centroids.py

```diff
@@ -62,10 +63,46 @@ def test_centroids_from_mean_verts_scrip(self):
         expected_face_x = uxgrid.face_lon.values
         expected_face_y = uxgrid.face_lat.values
 
-        _populate_centroid_coord(uxgrid, repopulate=True)
+        _populate_face_centroids(uxgrid, repopulate=True)
 
         computed_face_x = (uxgrid.face_lon.values + 180) % 360 - 180
         computed_face_y = uxgrid.face_lat.values
 
         nt.assert_array_almost_equal(expected_face_x, computed_face_x)
         nt.assert_array_almost_equal(expected_face_y, computed_face_y)
+
+    def test_edge_centroids_from_triangle(self):
+        """Test finding the centroid of a triangle."""
+        # Create a triangle
+        test_triangle = np.array([(0, 0, 0), (-1, 1, 0), (-1, -1, 0)])
+
+        # Open the dataset and find the centroids
+        grid = ux.open_grid(test_triangle)
+        _populate_edge_centroids(grid)
+
+        # compute edge_xyz for first edge
+        centroid_x = np.mean(grid.node_x[grid.edge_node_connectivity[0][0:]])
+        centroid_y = np.mean(grid.node_y[grid.edge_node_connectivity[0][0:]])
+        centroid_z = np.mean(grid.node_z[grid.edge_node_connectivity[0][0:]])
+
+        # Test the values of computed first edge centroid and the populated one
+        self.assertEqual(centroid_x, grid.edge_x[0])
+        self.assertEqual(centroid_y, grid.edge_y[0])
+        self.assertEqual(centroid_z, grid.edge_z[0])
+
+    def test_edge_centroids_from_mpas(self):
+        """Test computed centroid values compared to values from a MPAS
+        dataset."""
+
+        uxgrid = ux.open_grid(mpasfile_QU)
+
+        expected_face_x = uxgrid.edge_lon.values
+        expected_face_y = uxgrid.edge_lat.values
+
+        _populate_edge_centroids(uxgrid, repopulate=True)
+
+        computed_face_x = (uxgrid.edge_lon.values + 180) % 360 - 180
+        computed_face_y = uxgrid.edge_lat.values
+
+        nt.assert_array_almost_equal(expected_face_x, computed_face_x)
+        nt.assert_array_almost_equal(expected_face_y, computed_face_y)
```

Comment:
A couple notes about the variables here
1. It should be `edge_` and not `face_`
2. Use `edge_lon` and `edge_lat` instead of `x` and `y`